src/Utils/PipedSpawn.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {spawn, SpawnOptions, spawnSync, SpawnSyncOptions} from 'child_process';
+
+import {Logger} from './Logger';
+
+/**
+ * Function to run '$ cmd1 | cmd2'
+ * @param cmd1 first command before pipe
+ * @param cmd1List args of cmd1
+ * @param cmd2 second command after pipe
+ * @param cmd2List args of cmd2
+ * @returns ChildProcess spawned cmd2
+ * @example
+ * grep = pipedSpawn('ls', ['-hl', '/'], {cwd: '/'}, 'grep', ['usr'], {});
+ * grep.stdout.on('data', (data)=>{ console.log(data.toString(); })
+ * grep.stderr.on('data', (data)=>{ console.log(data.toString(); })
+ * grep.on('exit', (exitcode: number | null, signal: NodeJS.Signals | null)=>{
+ *   if (exitcode === null) { // handle signal
+ *   } else if (exitcode === 0) { console.log("success!");
+ *   } else { console.log("exit code is not 0"); })
+ */
+export function pipedSpawn(
+    cmd1: string, cmd1List: string[], cmd1Option: SpawnOptions, cmd2: string, cmd2List: string[],
```

Comment:
how about taking arguments as : `string[] and SpawnOptions`..? 
```suggestion
    cmd1: { cmdstrs: string[], options: SpawnOpotions}, cmd2: { cmdstrs: string[], options: SpawnOpotions}...
```

Because 'cmd1' is the command string which comes first and cmd1List is others, you can simplify the args.

Reply:
Thanks for the suggestion.
Actually, this follows the parameter of `spawn(cmd: string, args: string[], option: SpawnOption)`. So I'd like to stick to this convention. :-)
Plus, `pipedSpawn()` will call two `spawn()` internally. If we use `cmd1: { cmdstrs: string[]...` we need to construct param (cmd string and arg of string[]) for `spawn` again.

Reply:
Sure, I understand your intention :-D